src/ECS/EventManager/EventManager.cpp

```diff
@@ -26,12 +29,10 @@ void EventManager::updateEvents()
     }
 }
 
-bool EventManager::checkEvent(int event)
+auto EventManager::checkEvent(int event) -> bool
```

Comment:
jamais

Reply:
https://stackoverflow.com/questions/52103775/advantage-of-using-trailing-return-type-in-c11-functions

Reply:
https://danielsieger.com/blog/2022/01/28/cpp-trailing-return-types.html